test/func/postman/ttk_func_collection.json

```diff
@@ -2948,7 +2948,7 @@
 									"    pm.response.to.have.status(202);",
 									"",
 									"",
-									"    pm.sendRequest(\"http://localhost:5050/longpolling/callbacks/parties/MSISDN/\"+pm.variables.get(\"pathfinderMSISDN\"), function (err, response) {",
+									"    pm.sendRequest(\"http://mojaloop-testing-toolkit:5050/longpolling/callbacks/parties/MSISDN/\"+pm.variables.get(\"pathfinderMSISDN\"), function (err, response) {",
```

Comment:
I do feel like these should be paramaratized, but that should probably be a future story as it out of scope for this PR.

Reply:
Yes agreed